### ISSUE TITLE
exporting/importing Service objects between clusters does not work in some cases

### ISSUE THREAD

**Issue by joemiller** [Kyrremann/kubectl-eksporter#3]
Hello,

I attempted to use the util to migrate a Service object from one cluster to another but the apply failed due to the `clusterIP` being exported. The IP was in a range not available on the new cluster.

```console
$ ku eksporter service -lapp=statuspage-checker | ku apply -f - -n production --context gke_pantheon-internal_us-central1_general-01
The Service "statuspage-checker" is invalid: spec.clusterIP: Invalid value: "10.191.253.144": provided IP is not in the valid range. The range of valid IPs is 10.20.16.0/20
```

The exported Service object looked like:

```yaml
---
apiVersion: v1
kind: Service
metadata:
  labels:
    app: statuspage-checker
    name: statuspage-checker
  name: statuspage-checker
spec:
  clusterIP: 10.191.253.144
  externalTrafficPolicy: Cluster
  ports:
  - nodePort: 32586
    port: 80
    protocol: TCP
    targetPort: 3000
  selector:
    app: statuspage-checker
    build: '124'
    state: serving
  sessionAffinity: None
  type: LoadBalancer
```

I also expected it might fail due to including the assigned nodePort's but the target cluster accepted them. I only needed to `grep -v clusterIP` out of the export.

Would you consider special handling for Service objects? Possibly via a -flag. I'm thinking that removing the clusterIP at least, and possibly the nodePorts (although there's probably cases where some folks would want to keep the nodePorts.) I might be able to make a PR if/when I have some time.

**Comment by Kyrremann** [comment]
No problem with adding more features. The easiest would just be to check if it's there, and if, delete it. That's mostly my usage of this plugin.

Maybe a more robust future would be to add a `--clean` flag where you specify what kind of resource it is, and then we delete specific things for that resource?

**Comment by joemiller** [comment]
Went with the straightforward route for now - https://github.com/Kyrremann/kubectl-eksporter/pull/4

**Comment by joemiller** [comment]
released in v1.3.0, closing. Although @porkloin and I ran into a bug caused by my implementation. PR to address that coming soon
